api/src/main/resources/liquibase.xml

```diff
@@ -195,4 +195,11 @@
 			DELETE FROM person_attribute WHERE value IS NULL OR value = '';
 		</sql>
 	</changeSet>
+	<changeSet id="ugandaemr-01-10-2018-1200" author="slubwama">
+		<comment>Move all Patients Ever enrolled in Facility Based Individual Management DSDM </comment>
+		<sql>
```

Comment:
@slubwama Add a pre-condition that checks whether the programs have been installed (by metadata deploy) add OnFail="CONTINUE" so that it can be executed on the next liquibase run, otherwise it fails on a new database without the programs that are loaded after liquibase is run